### DISCUSSION TITLE
Show items in Flamtaer bag

### DISCUSSION THREAD

**Discussion by boomer00oo** [runelite/runelite#15031] [Ideas]
**Is your feature request related to a problem? Please describe.**
I am training construction with the Flamtaer back and limestone bricks to save money. It would make this more efficient if I could see how much was in the bag. Would be nice if it acted like a rune pouch.

**Describe the solution you'd like**
The rune pouch information, but for the Flamtaer bag.

**Describe alternatives you've considered**
N.a.
**Additional context**
https://i.imgur.com/pavjkYU.jpg

**Comment by trelkins3** [comment]
@Nightfirecat @deathbeam - if I wanted to implement this do you think the best way is to implement a new plugin (ex. FlamtaerBagOverlay.java) that's very similar to RunepouchOverlay.java and extends WidgetItemOverlay? Or would it be better to nestle the code somewhere else instead of creating a new plugin?

**Comment by Nightfirecat** [comment]
It wouldn't fit into any existing plugin, so it would be better suited for a separate plugin.

**Comment by trelkins3** [comment]
Flamtaer bag doesn't use varbits, so this may be significantly harder than implementing something like the rune pouch.

**Comment by robrichardson13** [comment]
Bumping this request.

This would be valuable even if it just showed how many items were in the bag.
